src/Stryker.CLI/README.md

```diff
@@ -0,0 +1,49 @@
+# Stryker.NET Command Line Interface
+The Stryker CLI is currently the only implemented way to use Stryker.NET. 
+
+## Getting started
+To install Stryker.NET on your *test project* add the following lines to your `.csproj` file.
+
+``` XML
+<ItemGroup>
+    <DotNetCliToolReference Include="dotnet-stryker" Version="0.1.*" />
+    <PackageReference Include="dotnet-stryker" Version="0.1.*" />
+</ItemGroup>
+```
+
+After adding the references, install the packages by executing `dotnet restore` inside the project folder.
+
+## Usage
+To kick off stryker, execute the following command:
+
+`dotnet stryker`
+
+## Configuration
+While Stryker.NET wants to be a non configuration needed tool, some settings are configurable.
+
+#### Specify your project to mutate
+When Stryker finds two or more project references inside your test project, it needs to know what project should be mutated. Pass the name of this project using:
+
+`dotnet stryker --project SomeProjectName.csproj`
```

Comment:
If this command is executed in the test project, I would expect the project to be `..\SomeProjectName.csproj`